src/core_plugins/kibana/public/assets/play-circle.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="8" height="8" viewBox="0 0 8 8">
+  <path d="M4 0c-2.21 0-4 1.79-4 4s1.79 4 4 4 4-1.79 4-4-1.79-4-4-4zm-1 2l3 2-3 2v-4z" />
+</svg>
```

Comment:
Deleting specific plugins was actually a strategy some folks used in the past, though it's less common now that it requires modifications to the code as well. Even though they're "core", I think it's best to avoid having common components rely on assets inside a plugin. It's weird to me that the logout icon is in that folder as well, but the others make sense because they're apps provided by the kibana plugin.